generators/server/templates/src/main/java/package/config/ReactiveSecurityConfiguration.java.ejs

```diff
@@ -201,14 +266,86 @@ public class SecurityConfiguration {
             .pathMatchers("/api/account/reset-password/init").permitAll()
             .pathMatchers("/api/account/reset-password/finish").permitAll()
             <%_ } _%>
+            .pathMatchers("/api/auth-info").permitAll()
             .pathMatchers("/api/**").authenticated()
             .pathMatchers("/management/health").permitAll()
             .pathMatchers("/management/info").permitAll()
             .pathMatchers("/management/prometheus").permitAll()
             .pathMatchers("/management/**").hasAuthority(AuthoritiesConstants.ADMIN);
+        <%_ if (authenticationType === 'oauth2') { _%>
+
+        http<%_ if (['monolith', 'gateway'].includes(applicationType)) { _%>.oauth2Login()
+            .authenticationSuccessHandler(this::onAuthenticationSuccess)
```

Comment:
@mraible will you also do the authenticationErrorHandler ?

Reply:
There's no reason to do one for OAuth 2.0 because the authentication will fail at the IdP and you won't be redirected back to the app.

Reply:
So we can't have the security audits for failed auth ?

Reply:
No, I don't think so. At least not in JHipster. They should be available in the IdP though.